src/main/java/de/blau/android/dialogs/BookmarkDialogs/BookmarkListAdapter.java

```diff
@@ -0,0 +1,128 @@
+package de.blau.android.dialogs.BookmarkDialogs;
```

Comment:
Packages should be lower case, and you seem to have moved the plural s to the wrong place (there is only on dialog here). I would simply name the package de.blau.android.dialogs.bookmarks